### PR TITLE
remove wrap token functionality

### PR DESCRIPTION
**Pull request by karthikeyan-cere**
image

### CODE REVIEW

File: src/chainbridgeConfig.ts [hunk 1]
```diff
@@ -50,78 +50,35 @@ export type ChainbridgeConfig = {
...
+      chainId: 5,
+      networkId: 5,
       name: "Ethereum",
       decimals: 18,
-      bridgeAddress: "0x478ab279Ac5F4bd69382D34cF2382606E6208eFc",
-      erc20HandlerAddress: "0x3483c3a1Af5e78AE5AaB07de3Ea57b6F3877745F",
-      rpcUrl: "wss://kovan.infura.io/ws/v3/e199aa0da7e54bd9be94de96ea753127",
+      bridgeAddress: "0xa806cA3bD88F790744462cBC34c40EDd5b8dc2Dd",
+      erc20HandlerAddress: "0xf934Bfc8B5241b6C9e0DfC9A329AD687e79c5498",
+      rpcUrl: "https://goerli.infura.io/v3/2532f2073c2d4a34b5b9544c76a3de8b",
```
Comment (AndreiNavoichyk) [inline src/chainbridgeConfig.ts]: Which Infura is using here? We should create a separate project for it

File: src/chainbridgeConfig.ts [hunk 2]
```diff
@@ -50,78 +50,35 @@ export type ChainbridgeConfig = {
...
-    //       symbol: "DOT",
-    //       resourceId: "substrate-native",
-    //     },
-    //   ],
-    // },
     {
-      chainId: 0,
-      networkId: 42,
+      chainId: 5,
+      networkId: 5,
```
Comment (AndreiNavoichyk) [inline src/chainbridgeConfig.ts]: Why 5?

File: src/chainbridgeConfig.ts [hunk 3]
```diff
@@ -50,78 +50,35 @@ export type ChainbridgeConfig = {
...
       name: "Ethereum",
       decimals: 18,
```
Comment (AndreiNavoichyk) [inline src/chainbridgeConfig.ts]: Is it native token decimals? If no, update with 10

File: src/chainbridgeConfig.ts [hunk 4]
```diff
@@ -50,78 +50,35 @@ export type ChainbridgeConfig = {
...
       nativeTokenSymbol: "ETH",
       tokens: [
         {
-          address: "0x2726A258f88b4e5B3a251e3d91594c527E10494D",
-          name: "wCFG",
-          symbol: "wCFG",
+          address: "0xb8A59CEF67d12C5C75836aEfF1d97943F5A9F662",
+          name: "CN",
+          symbol: "CS",
           imageUri: ETHIcon,
```
Comment (AndreiNavoichyk) [inline src/chainbridgeConfig.ts]: Update icon with Cere

Comment (karthikeyan-cere) [reply src/chainbridgeConfig.ts]: Moved config changes to seperate PR

File: src/chainbridgeConfig.ts (hunk 3 above)
Comment (karthikeyan-cere) [reply src/chainbridgeConfig.ts]: Moved config changes to seperate PR

File: src/chainbridgeConfig.ts (hunk 2 above)
Comment (karthikeyan-cere) [reply src/chainbridgeConfig.ts]: Moved config changes to seperate PR

File: src/chainbridgeConfig.ts (hunk 1 above)
Comment (karthikeyan-cere) [reply src/chainbridgeConfig.ts]: Moved config changes to seperate PR

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**